src/templates/default/styles/index.css

```diff
@@ -7,11 +7,22 @@
 
 .mk-general-item-title {
   margin: 0;
-  color: #333;
+  color: #336;
```

Comment:
Please don't change the main color of titles. I will prepare the feature to customize the template, not just the colors!